conda_smithy/schema.py

```diff
@@ -0,0 +1,1141 @@
+                # built packages) as an Azure pipeline artifact that can be downloaded
+                store_build_artifacts: False
+                # toggle for freeing up some extra space on the default Azure Pipelines
+                # linux image before running the Docker container for building
+                free_disk_space: False
+                # limit the amount of CI jobs running concurrently at a given time
+                # each OS will get its proportional share of the configured value
+                max_parallel: 25
+
+
+        .. _self-hosted_azure-config:
+
+        Below is an example configuration for setting up a self-hosted Azure agent for Linux:
+
+        .. code-block:: yaml
+
+            azure:
+                settings_linux:
+                pool:
+                    name: your_local_pool_name
+                    demands:
+                    - some_key -equals some_value
+                workspace:
+                    clean: all
+                strategy:
+                    maxParallel: 1
+
+        Below is an example configuration for adding a swapfile on an Azure agent for Linux:
+
+        .. code-block:: yaml
+
+            azure:
+                settings_linux:
+                    swapfile_size: 10GiB
+        """,
+    )
+
+    drone: Optional[Dict[str, str]] = Field(
+        default_factory=dict,
+        description="""
+        Drone CI settings. This is usually read-only and should not normally be
+        manually modified. Tools like conda-smithy may modify this, as needed.
+        """,
+    )
+
+    github_actions: Optional[GithubActionsConfig] = Field(
+        default_factory=GithubActionsConfig,
+        description="""
+        GitHub Actions CI settings. This is usually read-only and should not normally be
+        manually modified. Tools like conda-smithy may modify this, as needed.
+        """,
+    )
+
+    woodpecker: Optional[Dict[str, str]] = Field(
+        default_factory=dict,
+        description="""
+        Woodpecker CI settings. This is usually read-only and should not normally be
+        manually modified. Tools like conda-smithy may modify this, as needed.
+        """,
+    )
+
+    ###################################
+    ####       Deprecated          ####
+    ###################################
+
+    # Deprecated values, only present for validation will not show up in
+    # the model dump, due to exclude=True
+
+    build_with_mambabuild: Optional[bool] = Field(
+        default=True,
+        exclude=True,
+        deprecated=True,
+        description="""
+        build_with_mambabuild is deprecated, use conda_build_tool instead. Configures the conda-forge CI to run a debug build using the ``mamba`` solver.
+        More information can be found in the
+        `mamba docs <https://conda-forge.org/docs/maintainer/maintainer_faq.html#mfaq-mamba-local>`__.
+
+        .. code-block:: yaml
```

Comment:
Hm, now that the site uses Markdown, we might need to update these RST blobs. I can do that, just noting it here for reference.